src/modlunky2/sprites/items.py

```diff
@@ -94,8 +95,11 @@ class ItemSheet(BaseSpriteLoader):
         "crossbow_empty_held": (2, 4, 3, 5),
         "crosbow_loaded_held": (3, 4, 4, 5),
         "metal_arrow": (4, 4, 5, 5),
+        "arrow_metal": (4, 4, 5, 5),
         "wood_arrow_poisoned": (5, 4, 6, 5),
         "metal_arrow_poisoned": (6, 4, 7, 5),
+        "arrow_wooden_poison": (5, 4, 6, 5),
+        "arrow_metal_poison": (6, 4, 7, 5),
```

Comment:
all of these (and `venom`) need to be added to `VALID_TILE_CODES` in `src\modlunky2\levels\tile_codes.py`

Reply:
oops

Reply:
done